include/Core/Config.php

```diff
@@ -123,24 +122,38 @@ public static function get($family, $key, $default_value=null, $refresh = false)
 	 *  The value to store
 	 * @return mixed Stored $value or false if the database update failed
 	 */
-	public static function set($family,$key,$value) {
+	public static function set($family, $key, $value) {
 		global $a;
 
+		$stored = self::get($family, $key);
+
+		if ($stored == $value) {
+			return true;
+		}
+
 		$a->config[$family][$key] = $value;
 
 		// manage array value
-		$dbvalue = (is_array($value)?serialize($value):$value);
+		$dbvalue = (is_array($value) ? serialize($value):$value);
```

Comment:
Can you please choose either:
- wrap the colon in spaces
- leave the question mark unwrapped?